test/test_sensors.py

```diff
@@ -156,3 +152,34 @@ def test_collisions():
             prev_collisions = collisions
 
     env.close()
+
+
+def test_static_pointgoal_sensor():
+    config = get_config()
+    if not os.path.exists(config.SIMULATOR.SCENE):
+        pytest.skip("Please download Habitat test data to data folder.")
+    config.defrost()
+    config.TASK.SENSORS = ["STATIC_POINTGOAL_SENSOR"]
+    config.freeze()
+    env = habitat.Env(config=config, dataset=None)
+    random.seed(123)
+    np.random.seed(123)
+
+    env.episodes = [
+        NavigationEpisode(
+            episode_id="0",
+            scene_id=config.SIMULATOR.SCENE,
+            start_position=[-1.3731, 0.08431, 8.60692],
+            start_rotation=[0, -np.pi, 0, np.pi],
+            goals=[NavigationGoal([-1.2731, 0.08431, 8.60692])],
+        )
+    ]
+
+    obs = env.reset()
+    for _ in range(5):
+        env.step(np.random.choice([0, 1, 2]))
```

Comment:
This approach of choosing actions will break if we change action space.  Check other tests to select non stop actions.